media/css/eoy.css

```diff
@@ -294,68 +475,160 @@ main {
   grid-row: 4 / span 1;
 }
 
+#hover-hearts {
+  position: absolute;
+  top: 30px;
+  right: 36px;
+  display: none;
+  gap: 3px;
+}
+
+#hover-heart-1,
+#hover-heart-2,
+#hover-heart-3 {
+  display: flex;
+  align-items: end;
+  opacity: 0;
+}
+
+#hover-heart-1 > svg {
+  animation-delay: 0.6s;
+}
+
+#hover-heart-3 {
+  animation-delay: 0.8s;
+}
+
+#hover-heart-1 > svg {
+  transform: rotate(-45deg);
+  width: 48px;
+  height: 48px;
+}
+
+#hover-heart-2 > svg {
+  width: 64px;
+  height: 64px;
+  transform: translateY(-9px);
+}
+
+#hover-heart-3 > svg {
+  transform: rotate(45deg);
+  width: 48px;
+  height: 48px;
+}
+
+.donate-banner:hover #hover-hearts {
+  display: flex;
+}
+
+.donate-banner:hover #hover-hearts > #hover-heart-1 {
+  animation: 2s heart-fade forwards ease-in-out;
+  animation-delay: 0.4s;
+}
+
+.donate-banner:hover #hover-hearts > #hover-heart-2 {
+  animation: 2s heart-fade forwards ease-in-out;
+  animation-delay: 0.3s;
+}
+
+.donate-banner:hover #hover-hearts > #hover-heart-3 {
+  animation: 2s heart-fade forwards ease-in-out;
+  animation-delay: 0.5s;
+}
```

Comment:
cute animation, I like it.
Let's make it a bit faster with each animation at 1s speed, and reduce the delay to 0.1s for heart-1 and 0.2s for heart-3, heart-2 doesn't need any delay.